src/components/project-logic.js

```diff
@@ -0,0 +1,89 @@
+import * as el from './dom-elements';
+import store from './local-storage';
+import { populateDom, populateDomByProject } from './populate-dom';
+
+class Project {
+  constructor(title) {
+    this.title = title;
+  }
+}
+
+function createDefaultProject() {
+  if (localStorage.getItem('localProjects') === null) {
+    const myDefaultProject = new Project('House chores');
+    el.projects.push(myDefaultProject);
+
+    store.addProjectToStore(myDefaultProject);
+    const projectOpt = document.createElement('option');
+    projectOpt.setAttribute('value', el.projects[0].title);
+    projectOpt.textContent = `${el.projects[0].title} (Default)`;
+    el.projectTitles.appendChild(projectOpt);
+    el.projectTitles.options[
+      el.projectTitles.selectedIndex
+    ].defaultSelected = true;
+  }
+}
```

Comment:
Use arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)) This applies to all your functions in other modules as well.